public/javascripts/data/user.js

```diff
@@ -2,24 +2,25 @@ const Database = require('better-sqlite3');
 const hashing = require('../hashing')
 
 
-
+// Function checks if user inputted username is available
 exports.signUp = (username) => {
     
     const db = new Database('public/javascripts/data/stories.db');
     const userStmt = db.prepare('SELECT user_name FROM users WHERE user_name = (?)');
```

Comment:
This userStmt will not protect against case-insensitive usernames. Investigate ways to do case insensitive text checks in sqlite. You may find `ILIKE` or `lower()` functions useful here

Reply:
For instance, I can sign up in the app right now with the username "John" as well as the username "jOhN"